graphql-codegen/codegen.ts

```diff
@@ -0,0 +1,52 @@
+import { addTypenameSelectionDocumentTransform } from '@graphql-codegen/client-preset'
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  schema: 'http://127.0.0.1:54321/graphql/v1',
```

Comment:
At first, Supabase has to be started to load schema from `graphql-codegen`.

Don't forget it! 👇

```bash
cd ~/tabi-memo/backend
supabase start
```

Reply:
To think about prod deployment, we should use env file for this URL?

Reply:
Exactly! I fixed it 👍
 https://github.com/tabi-memo/frontend/pull/6/commits/d241b0369a37857d5d969fd9f498ef4deedaf891